### PR TITLE
Use wayland if the system supports it

### PR DESCRIPTION
**Pull request by xuhdev**
Change "--socket=x11" to "--socket=fallback-x11"

### CODE REVIEW

File: net.sourceforge.scidvspc.scidvspc.yml [hunk 1]
```diff
@@ -7,10 +7,10 @@ rename-icon: scid #128x128
 
 finish-args:
   - --filesystem=xdg-documents
-  - --share=network     
-  - --socket=x11           
-  - --share=ipc         
-  - --device=dri         
+  - --share=network
+  - --socket=fallback-x11
```
Comment (xuhdev) [inline net.sourceforge.scidvspc.scidvspc.yml:11]: If I add `--socket=wayland`, the program won't start. But if I only specify `fallback-x11`, the program can start as a wayland app. I'm not sure what to do here.

Comment (bztd) [reply net.sourceforge.scidvspc.scidvspc.yml:11]: As far as I understand tk/tcl does not support Wayland.

Comment (bztd) [reply net.sourceforge.scidvspc.scidvspc.yml:11]: As far as I understand, tk/tcl doesn't support Wayland. I assume that by using "--socket=fallback-x11", you're using the X11 integration that Flatpak offers in Wayland.

The program crashes if "--socket=x11" is specified on your PC?

Comment (xuhdev) [reply net.sourceforge.scidvspc.scidvspc.yml:11]: I agree -- actually it turns out something's strange about my system, where I can run scid vs pc (either with or without flatpak) native on Wayland, but it's not supposed to. Thanks for looking into this!

Comment (bztd) [reply net.sourceforge.scidvspc.scidvspc.yml:11]: Yes, it's magic.

**Pull request closed without merging**